src/components/messages/MsgDetails/index.tsx

```diff
@@ -49,6 +53,8 @@ const Msg = ({ message }: { message: Message['message'] }): ReactElement => {
 
 const MsgDetails = ({ msg }: { msg: Message }) => {
   const wallet = useWallet()
+  const isWrongChain = useIsWrongChain()
+  const isConfirmed = msg.status === MessageStatus.CONFIRMED
```

Comment:
Is there a reason we are not using the same logic for `isConfirmed` here as in the other component i.e.

```
const confirmationsNeeded = msg.confirmationsRequired - msg.confirmationsSubmitted
const isConfirmed = confirmationsNeeded <= 0
```

Reply:
This was copied logic from `TxSigners`. I've adjusted it to reference the status.